### ISSUE TITLE
[BadDomList] make tests self-contained (Trac #2005)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1997]
Calling I3Live from a unit test is probably a bad idea.  Not only can it fail (1), it also puts unnecessary load on I3Live.  Consider mocking the call and returning cached output.  This is also a good way to test what the code does with a failure.

(1) http://builds.icecube.wisc.edu/builders/Ubuntu%2014.04%20%28cvmfs%29/builds/2087/steps/test_1/logs/stdio

Migrated from https://code.icecube.wisc.edu/ticket/2005
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:44", 
    "description": "Calling I3Live from a unit test is probably a bad idea.  Not only can it fail (1), it also puts unnecessary load on I3Live.  Consider mocking the call and returning cached output.  This is also a good way to test what the code does with a failure.\n\n(1) http://builds.icecube.wisc.edu/builders/Ubuntu%2014.04%20%28cvmfs%29/builds/2087/steps/test_1/logs/stdio\n", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067284370534", 
    "component": "combo reconstruction", 
    "summary": "[BadDomList] make tests self-contained", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-05-08T01:12:44", 
    "milestone": "", 
    "owner": "joertlin", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170508T14:55:09: 
* joertlin changed status from "new" to "accepted"
* joertlin changed owner from "olivas" to "joertlin"

**Comment by icecube-trac** [comment]
Trac update at 20170825T19:13:17: 
* joertlin changed resolution from "" to "fixed"
* joertlin commented:

> In changeset 157812:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="157812"
> * Bugfix
> * Made tests self-contained
> 
> Closes #1997
> ```


* joertlin changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:44: 
* olivas commented:

> Milestone July 2017 deleted


* olivas changed milestone from "July 2017" to ""
